src/utils/FormInput.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+interface FormInputProps{
+    classes:string,
+    id:string
+}
+
+const FormInput = ({classes,id}:FormInputProps) => {
+  return (
```

Comment:
You can use <label> component here. So whenever you use input, label comes along with it, you dont have to hastle adding label for every input component.
Also you can make label optional for those input elements which dont need labels